config.yaml

```diff
@@ -92,16 +92,20 @@ rabbitmq:
 
   # A list of cluster members, including network-specific IPs.
   # The 'default' network IP must be set for each member.
-  # If installing a single external rabbit (not intended to be part of a cluster),
-  # or an all-in-one manager this section can be left blank.
+  # If installing an all-in-one manager this section can be left blank.
   # Example:
   # cluster_members:
   #   <hostname of rabbit node>:
-  #     default: <ip of rabbit node> (not needed if node name is resolvable via DNS)
-  #     <other network name>: <ip for this node on 'other network'>
+  #     node_id: <Cloudify's auto-generated id of the rabbit node> (only needed on
```

Comment:
If it's auto-generated and not user controlled, why is it in the config?

Reply:
Good question, I'm not sure how to phrase it right.
The node_id is auto-generated when installing the status reporter on the rabbit/postgres node and the user has to take it and add it here as he adds the IP and hostname.